### PR TITLE
Fix "toggle_type" column of Setting toggles in state report

### PR DESCRIPTION
**Pull request by regisb**
**Description:** The toggle_type column used to be "SettingToggle" and "SettingDictToggle" for
these classes. It should instead be "django_settings".

**JIRA:** https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation

**Merge deadline:** Feb 24 2021

**Reviewers:**
- [ ] @robrap 

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [ ] ~~Version bumped~~
- [x] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [ ] ~~Create a tag~~
- [ ] ~~Check new version is pushed to PyPi after tag-triggered build is finished.~~
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** :warning: THIS CHANGE IS UNTESTED. I have no idea how to generate sample json/yml reports. The project README mentions that:

> Each IDA exposes an HTTP endpoint provided by the [TODO: pending] edx-toggles Django app. A scheduled job can use an OAuth client associated with a staff user to get a JWT, and then use the JWT against each configured IDA's toggle state endpoint.

Unfortunately I have not taken the time to perform these steps. It would be great if someone could either test these changes or provide me with a sample toggle report.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @regisb! I've created [OSPR-5360](https://openedx.atlassian.net/browse/OSPR-5360) to keep track of it in JIRA.

As a core committer in this repo, you can merge this once the pull request is approved per the [core committer reviewer requirements](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Open+edX+Core+Committers#Reviews-and-Ownership) and according to the agreement with your edX Champion.

**Comment by natabene** [comment]
@regisb Thank you for your contribution. @robrap I assume this is for you to review.

**Review by robrap — APPROVED** [review]
Let’s just merge and see. 🙂

**Comment by robrap** [comment]
@regisb: Note: feel free to release with other changes later and maybe a brief bullet in the change log?

**Comment by regisb** [comment]
> Note: feel free to release with other changes later and maybe a brief bullet in the change log?

Thanks! I added a changelog entry in the "unreleased" section.

**Comment by edx-webhook** [comment]
@regisb 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-toggles&entry.867055334=https://github.com/edx/edx-toggles/pull/109&entry.1484655318=https://github.com/regisb&entry.752974735=2021-01-13+14:53&entry.1917517419=2021-01-14+08:07&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@jmbowman, @kdmccormick: thought you might like to know that regisb merged this pull request.

**Pull request merged**
